src/schema/imported/manifest.json

```diff
@@ -124,10 +124,13 @@
                 },
                 "incognito": {
                   "type": "string",
+                  "description": "The 'split' value is not supported.",
                   "enum": [
                     "not_allowed",
-                    "spanning"
+                    "spanning",
+                    "split"
                   ],
+                  "postprocess": "incognitoSplitUnsupportedAndFallback",
```

Comment:
We should probably fix https://github.com/mozilla/addons-linter/issues/5261 while we're at it. WDYT?